Rules/CTF/mapcycle.cfg

```diff
@@ -16,11 +16,14 @@ mapcycle =
 			Maps/CTF/bunnie_Luthadel.png;
 				#Ej
 			Maps/CTF/Ej_Cataracts.png;
+			Maps/CTF/Ej_Chambers.png;
 				#Ferrezinhre
 			Maps/CTF/Ferrezinhre_Highlands.png;
 				#Geti
 			Maps/CTF/HearthPlains.png;
 			Maps/CTF/Snatchmark.png;
+				#ffsff
+			Maps/CTF/PlatformFlats.png;
```

Comment:
Wrong file name.

Reply:
Please merge and fix? You know I don’t have PC access.